CONTRIBUTING.md

```diff
@@ -508,6 +508,47 @@ In case you need to switch to sessionType=cookie, you
 - Use `AUTHGEAR_WEB_SDK_SESSION_TYPE=cookie` in your .env
 - Access the portal at port 8001 or 8011
 
+## Switch to Database config source
+
+1. In your `.env` set these values:
+
+```
+CONFIG_SOURCE_TYPE=database
+CUSTOM_RESOURCE_DIRECTORY=./hack/custom-resources
+```
+
+2. Create a row in `_portal_config_source`:
+
+```sql
+INSERT INTO "public"."_portal_config_source"("id","app_id","created_at","updated_at","data","plan_name")
+VALUES
+('00000000-0000-0000-0000-000000000000','accounts',NOW(),NOW(),'{{ YOUR_DATA }}','free');
+```
+
+Note the `data` column should be obtained by running the following command:
+```
+go run ./cmd/portal internal configsource pack -i ./var
+```
```

Comment:
I think Step 2 is actually line 205: `go run ./cmd/portal internal configsource create ./var`

Reply:
I think these steps are not necessary when you use local config source, so maybe I move them to the new "Switch to Database config source" section?